uaclient/cli/enable.py

```diff
@@ -86,6 +94,32 @@ def prompt_for_dependency_handling(
                 required_service=required_service_title,
             )
 
+    if (
+        service in enabled_service_names
+        and variant
+        and variant not in enabled_service_variants
+    ):
+        variant_title = entitlements.get_title(cfg, service, variant)
+        enabled_variant_title = entitlements.get_title(
+            cfg, service, enabled_service_variants[0]
+        )
```

Comment:
variant names without the associated service name is missing information. There could be a "generic" variant for more than one service.

I suggest something like the following:
```suggestion
    variant_enabled = next(
        (s for s in enabled_services if s.name == service and s.variant_enabled and s.variant_name != variant), None
    )
    if variant_enabled is not None:
        to_be_enabled_title = entitlements.get_title(cfg, service, variant)
        enabled_variant_title = entitlements.get_title(cfg, service, variant_enabled.variant_name)
```

Reply:
Understood, the use of `next` is also better than the indexing